constrain/eprunner.py

```diff
@@ -76,7 +75,7 @@ def run_simulation(self):
 
         command.append(self.idf_path)
 
-        self.process_run = subprocess.run(command, shell=True, capture_output=True)
+        self.process_run = subprocess.run(command, shell=False, capture_output=True)
```

Comment:
https://stackoverflow.com/questions/3172470/actual-meaning-of-shell-true-in-subprocess